docker/php/supervisord.conf

```diff
@@ -10,4 +10,10 @@ command=apachectl -D "FOREGROUND" -k start
 command=/usr/local/bin/php /var/www/html/vendor/chrisboulton/php-resque/bin/resque
 autostart=true
 autorestart=true
-environment=APP_INCLUDE=/var/www/html/worker_bootstrap.php,INTERVAL=1,REDIS_BACKEND=redis:6379,QUEUE=bulk_label_offer
\ No newline at end of file
+environment=APP_INCLUDE=/var/www/html/worker_bootstrap.php,INTERVAL=1,REDIS_BACKEND=redis:6379,QUEUE=bulk_label_offer
```

Comment:
Did you rebuild the image?

Reply:
You can do that with `docker-compose up -d php --build` for just one container.

Reply:
Since the main issue of this ticket was to get the acceptance tests to run correctly on docker, and only `supervisord` is needed, a correct setup of `supervisorctl` is out of scope.